lib/pluginmanager/templates/filter-plugin/.travis.yml

```diff
@@ -0,0 +1,18 @@
+---
+sudo: false
+language: ruby
+cache: bundler
+matrix:
+  include:
+  - rvm: jruby-9.1.13.0
+    env: LOGSTASH_BRANCH=master
+  - rvm: jruby-9.1.13.0
+    env: LOGSTASH_BRANCH=6.x
+  - rvm: jruby-9.1.13.0
+    env: LOGSTASH_BRANCH=6.0
```

Comment:
For the template, I would suggest to leave off `6.0`.  I need to update (add or replace)  to `6.2` across the board.  for the template, I think master, 6.x, and 5.6 have a decent longevity.